batch/batches/ExportCsv/Engine/KObjectExportEngine.php

```diff
@@ -8,7 +8,7 @@ abstract class KObjectExportEngine
 {
 	/**
 	 * @param int $objectType of enum KalturaCopyObjectType
```

Comment:
@TalDubovKaltura  this was originally my bad, could I ask you to change enum type in this comment to KalturaExportObjectType?

Reply:
Done.